fe/fe-core/src/main/java/com/starrocks/connector/iceberg/io/IcebergCachingFileIO.java

```diff
@@ -120,6 +121,20 @@ public void initialize(Map<String, String> properties) {
         }
     }
 
+    @Override
+    public void close() {
```

Comment:
Thanks for your contribution!  To be honest, this is just a warnning log. it will help us to close the fileio when trigger gc once. GC will also play the same role when the cache triggers the expire operation. We did not actively shut down resources when we remove connector or remove query level metadata. And I found there is no way to selectively close fileIO in `GlueCatalog$fileIOCloser`. But we can close the all file io when dropping connector. This only  I wrote a simple shutdown test [demo](https://github.com/StarRocks/starrocks/pull/48002), you could refer to this patch to continue improving your PR, thanks you contribution again!

Reply:
i get tons of stacktraces with glue catalog without the patch.
it's really difficult to operate the cluster

and i think my memory usage has come down while refreshing the catalog

Reply:
okay. good job